src/test/java/uk/gov/justice/probation/courtcaseservice/controller/CourtCaseControllerIntTest.java

```diff
@@ -229,7 +229,7 @@ public void createCaseDataWithIncorrectCourt() {
                 .accept(ContentType.JSON)
                 .body(caseDetails)
                 .when()
-                .put("/case/" + CASE_ID)
+                .put("/case/" + "123456")
```

Comment:
Why lose the constant ? And there's no point concatenating. IDE mistake ?

Reply:
yep, will amend